src/app/components/all-products-components/all-products-content/aside/farms/farms.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { MatCheckboxChange } from '@angular/material/checkbox';
+import { FiltersService } from '../../../../../services/all-products/filters/filters.service';
+
+@Component({
+  selector: 'app-farms',
+  templateUrl: './farms.component.html',
+  styleUrls: ['./farms.component.css']
+})
+export class FarmsComponent implements OnInit {
+
+  public farmNames: string[] = [];
+  @Output() farmValue = new EventEmitter<MatCheckboxChange>();
+
+  constructor(private filtersService: FiltersService) {}
+
+  ngOnInit(): void {
+    this.getFarmsName();
+  }
+
+  getFarmsName(): void {
```

Comment:
also add access modifier for methods

Reply:
modifiers were added to methods and variables.